### ISSUE TITLE
[Regression] Cmd + click doesn't open a new window anymore

### ISSUE THREAD

**Issue by mathieutu** [wavebox/waveboxapp#892]
* Wavebox Version: last beta
* Operating System & Version: macOS 10.13.6

**Expected & actual behavior**
Until the last update, a <kbd>Cmd</kbd> + <kbd>click</kbd> opened the link in a new window in the same service, which was super convenient.

This behavior disappeared, and now, it opens the link in standard browser, and I have to right click, then "Open Link in Wavebox" then "Open Link in a New Windows", which is too bad...

Is it normal/ a wanted behavior or a fail on my side? Could we keep the old one?

Thanks.

**Comment by Ash258** [comment]
See: https://github.com/wavebox/waveboxapp/issues/878#issuecomment-447889416

**Comment by Thomas101** [comment]
As a rule of thumb, we try not to change default behaviours. We've had quite a lot of requests for #878 on support, but it was just the fact that we added a bunch of logic for other things that we were able to add 878 into the release quite quickly.

In this case our thinking was because Shift+Click & Cmd+Click do the same thing it would be helpful to have one of those behave differently by default. We chose not to change Shift+Click, only because Gmail offers this as the suggested method to popout compose windows.

Obviously, not everyone want's Cmd+Click to open pages in the browser - hence the option to change the behaviour :)
